### DISCUSSION TITLE
Auto kit settings

### DISCUSSION THREAD

**Discussion by brucmod** [tesla-android/issue-tracker#133] [Q&A]
Anybody know what many of the advanced settings do?  Anyone with a best practices list on settings for Tesla android?

**Comment by mikegapinski** [comment]
Audio source -> bluetooth
Framerate -> 30 or 60, that's personal preference
resolution -> middle preset
charge rate -> slow 
auto play -> disabled

That's how I use it

**Reply by brucmod** [reply]
There are a slew of advanced settings in autokit, have you played with any of them ?

**Reply by adamChrystie** [reply]
> Audio source -> bluetooth Framerate -> 30 or 60, that's personal preference resolution -> middle preset charge rate -> slow auto play -> disabled
> 
> That's how I use it

@brucmod  if you use wired apple carplay, you might  need to set charge rate to fast (high/normal) in order for carlinkit dongle to see phone. My newer carlinkit from 2/2022 (wired & wireless) requires the setting to be on fast (high/normal). My older carlinkit from 7/2021 (wired only) didn't have an issue when charge rate was set to slow. I personally use the wired/wireless dongle so I leave my charge rate at high/normal incase I want to use wired mode. I believe the setting controls how much power is passed through carlinkit dongle for charging the iPhone. Wish carlinkit / auto kit had better documentation.
